clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -210,7 +210,9 @@
 
     /** <code>request.timeout.ms</code> */
     public static final String REQUEST_TIMEOUT_MS_CONFIG = CommonClientConfigs.REQUEST_TIMEOUT_MS_CONFIG;
-    private static final String REQUEST_TIMEOUT_MS_DOC = CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC;
+    private static final String REQUEST_TIMEOUT_MS_DOC = CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC
+                                                        + " request.timeout.ms should be larger than value of broker side config replica.lag.time.max.ms"
```

Comment:
should be larger than value of broker side config replica.lag.time.max.ms => should be larger than replica.lag.time.max.ms, a broker side configuration ?